corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py

```diff
@@ -31,7 +31,7 @@ def handle(self, domain, **options):
         if not options['ignore_domain_in_use'] and is_domain_in_use(domain):
```

Comment:
shouldn't it be `options['ignore-domain-in-use-check']`?

Reply:
Reasonable question. If you look at where this argument is defined:
```
        parser.add_argument(
            '--ignore-domain-in-use-check',
            action='store_true',
            dest='ignore_domain_in_use',
            default=False,
            help='Allow deleting forms and cases for a domain that is in use.',
        )
```
You can see that the destination var is specified as `ignore_domain_in_use` 👍🏻

Reply:
That is still kinda gross that those two vars don't match though. I'll update that.

Reply:
No worries at all! It lead to fcf2d10e3c542bdf4e1512f208b9aa55aad66f12 which is better (I think)